e2e/test/scenarios/onboarding/setup/setup.cy.spec.ts

```diff
@@ -319,20 +319,20 @@ describe("scenarios > setup", () => {
 
     cy.location("pathname").should("eq", "/");
 
-    main().findByText("Embed Metabase in your app").should("exist");
-
-    main()
-      .findByRole("link", { name: "Learn more" })
-      .should("have.attr", "href")
-      .and(
-        "match",
-        /https:\/\/www.metabase.com\/docs\/[^\/]*\/embedding\/start\.html\?utm_media=embed-minimal-homepage/,
-      );
+    // main().findByText("Embed Metabase in your app").should("exist");
 
-    cy.reload();
+    // main()
```

Comment:
Temporarily commented this test as I removed the minimal version, will update the test when I'll add back the full homepage in a following PR

Reply:
Test added back here: https://github.com/metabase/metabase/pull/40587/files#diff-873ba50b242fadb813c5b50b247eae5052e5e46396450b8117a73cef3a6e0f7f